### DISCUSSION TITLE
Is there a proper way to install local plugin's dependencies?

### DISCUSSION THREAD

**Discussion by darron1217** [strapi/strapi#8469] [Legacy Topics]
I found the issue related to this question.
https://github.com/strapi/strapi/issues/2568

but I cannot find `npm run setup` command

Edit
I temporary fixed it by adding `postinstall` script on package.json.
Is it a proper way to do it?
```
{
  "scripts": {
    "postinstall": "npm i --prefix plugins/my-plugin1 && npm i --prefix plugins/my-plugin2"
  }
}
```

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

**Reply by darron1217** [reply]
Got it
